Assets/HoloToolkit/SpatialMapping/Materials/Wireframe.mat

```diff
@@ -72,7 +73,7 @@ Material:
     - _WireThickness: 100
     - _ZWrite: 1
     m_Colors:
-    - _BaseColor: {r: 0, g: 0, b: 0, a: 1}
+    - _BaseColor: {r: 0.0036224083, g: 0.24982113, b: 0.49264705, a: 1}
```

Comment:
Should we be editing the base material, or using a copy for our example?

May need to revert.

Reply:
Agreed. Why was the color changed?

Reply:
Oh, I wanted to update default white to more similar to what we have in Fragments (changed it to blue)
Original:
![2018-05-17 10_05_49-photos](https://user-images.githubusercontent.com/13754172/40192732-0c5b1dd2-59ba-11e8-8495-7fc9ae73f305.png)
Updated:
![2018-05-17 10_06_13-photos](https://user-images.githubusercontent.com/13754172/40192731-0c47f432-59ba-11e8-8612-2486f18ff191.png)

Reply:
I can duplicate with a new name 'WireframeBlue.mat' if we want. and assign it to SpatialMapping prefab as a default material.

Reply:
Please do duplicate the material. Changing the color can cause confusion with customers.

Thanks!